src/OXRS_Rack32.cpp

```diff
@@ -407,7 +407,7 @@ void OXRS_Rack32::begin(jsonCallback config, jsonCallback command)
   _screen.begin();
 
   // Display firmware details
-  _screen.draw_header(_fwMakerCode, _fwShortName, _fwVersion, "ESP32");
+  _screen.draw_header(_fwMaker, _fwShortName, _fwVersion, "ESP32");
```

Comment:
Pedantic, but can we make this method signature match the Rack32 constructor - i.e. `fwName, fwMaker, fwVersion, `ESP32`?